neusomatic/include/msa.hpp

```diff
@@ -44,8 +44,11 @@ class GappedSeq {
 public:
   GappedSeq(unsigned rlen): bases_(rlen), gaps_(rlen) {}
   
-  GappedSeq(const std::string& rseq, const GInv& ginv): bases_(rseq.begin(), rseq.end()), gaps_(rseq.length()), ginv_(ginv) {}
-
+  GappedSeq(const std::string& rseq, const GInv& ginv): bases_(rseq.begin(), rseq.end()), gaps_(rseq.length()), ginv_(ginv) {
+    if (rseq.size() + ginv.left()  != ginv.right()) {
+      throw std::runtime_error("seq length does not match the interval length");
```

Comment:
@msahraeian This happens if I didn't fix the bug. I don't think this will happen again after the fix. But if it happens, it will save us plenty of time for debugging, lol.

Reply:
sounds fine. thanks. I am running it on full dataset to see if it works as we expect.

Reply:
It seems to run smoothly on my test datasets. @marghoob can we merge it?